pkg/terraform/terraform.go

```diff
@@ -93,6 +89,16 @@ func unpackAndInit(dir string, platform string) (err error) {
 		return errors.Wrap(err, "failed to unpack Terraform modules")
 	}
 
+	err = data.Unpack(filepath.Join(dir, executablePath), filepath.FromSlash(executablePath))
+	if err != nil {
+		return errors.Wrap(err, "failed to unpack Terraform binary")
+	}
+
+	err = os.Chmod(filepath.Join(dir, executablePath), 0555)
```

Comment:
This will need `FromSlash` too, and so will `exec.Command(filepath.Join(clusterDir, executablePath), args...)`.  In fact, why are we bothering with `bin` at all?  This would be easier if we just had `executableFilename = "terraform", or some such.  And since we have CI to catch us if we typo an instance, I'd be fine hard-coding `"terraform"` everywhere and skipping the variable altogether (although I'm also fine with a slash-less `executableFilename`).